File: Pessoa.java
public class Pessoa {
    private final String nome;
    private final String curso;
    private final String campus;
    private final double nota;
    private final String tipoVaga;

    public Pessoa(String nome, String curso, String campus, double nota, String tipoVaga) {
        this.nome = nome;
        this.curso = curso;
        this.campus = campus;
        this.nota = nota;
        this.tipoVaga = tipoVaga;
    }

    public String getNome() {
        return nome;
    }

    public String getCurso() {
        return curso;
    }

    public String getCampus() {
        return campus;
    }

    public double getNota() {
        return nota;
    }

    public String getTipoVaga() {
        return tipoVaga;
    }
}

File: pythonproject/src/python/Pessoa.py
class Pessoa:
    def __init__(self, nome: str, curso: str, campus: str, tipoVaga: str, nota: float, posicao: int):
        self.__nome = nome
        self.__curso = curso
        self.__campus = campus
        self.__tipoVaga = tipoVaga
        self.__nota = nota
        self.__posicao = posicao
    
    def getNome(self) -> str:
        return self.__nome
    
    def getCurso(self) -> str:
        return self.__curso
    
    def getCampus(self) -> str:
        return self.__campus
    
    def getNota(self) -> float:
        return self.__nota
    
    def getTipoVaga(self) -> str:
        return self.__tipoVaga
    
    def getPosicao(self) -> int:
        return self.__posicao
    
    def __str__(self) -> str:
        return f"{self.__nome} passou em {self.__curso}"
